web/webkit/src/main/scala/net/liftweb/http/HtmlNormalizer.scala

```diff
@@ -0,0 +1,291 @@
+package net.liftweb
+package http
+
+import scala.xml._
+import scala.collection.immutable.Vector
+
+import js.JsCmd
+import js.JsCmds.Noop
+import js.JE.{AnonFunc, Call, JsRaw}
+
+import util.Helpers._
+
+/**
+ * Represents an HTML attribute for an event handler. Carries the event name and
+ * the JS that should run when that event is triggered as a String.
+ */
+private case class EventAttribute(eventName: String, jsString: String)
```

Comment:
Yeah, this is once it's extracted directly from HTML (I assume you meant `JsCmd` or `JsExp` rather than `JValue`), so there's no real need to reify it into a typed thing yet.